src/Controller/CartographieController.php

```diff
@@ -23,9 +23,17 @@ public function index(
             } else {
                 $date = new \DateTimeImmutable();
             }
+            $swLat = $request->get('swLat');
+            $swLng = $request->get('swLng');
+            $neLat = $request->get('neLat');
+            $neLng = $request->get('neLng');
```

Comment:
Faudrait mapper un DTO avec controles des données en entrée 
https://symfony.com/blog/new-in-symfony-6-3-mapping-request-data-to-typed-objects

![image](https://github.com/user-attachments/assets/33447711-7e3d-4ca5-9118-c99439e074ed)